unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Settings/SettingsModule.cs

```diff
@@ -4,14 +4,14 @@
 
 namespace DCL.SettingsCommon
 {
-    public class SettingsModule<T> where T : struct
+    public class SettingsModule<T> where T : ICloneable
```

Comment:
This reopens #1312, we have to analyze this later as it breaks the quality preset dropdown